sources/router.c

```diff
@@ -268,10 +269,10 @@ od_router_route(od_router_t *router, od_config_t *config, od_client_t *client)
 		is_shared = od_config_is_multi_workers(config);
 		route = od_route_pool_new(&router->route_pool, is_shared, &id, rule);
 		if (route == NULL) {
-			od_router_unlock(router);
-			return OD_ROUTER_ERROR;
-		}
-	}
+            od_router_unlock(router);
```

Comment:
I though we have automatic formatting...

Reply:
I forgot about it)